_posts/0000-01-02-eHills-F.md

```diff
@@ -0,0 +1,6 @@
+--
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
In the beginning was the code, and the code was with Ethan, and the code was Ethan.